### ISSUE TITLE
Dialog disappears on device rotation

### ISSUE THREAD

**Issue by reverse-unina** [ligi/PassAndroid#127]
When i perform these sequences of events, the dialog on screen disappears:

Sequence 1:
1. Swipe right on an item in list
2. Orientation Change (in this case the element reappears in list)

Sequence 2-4:
1. Click on an element in list
2. Click on Edit icon
3. Click on:
a. Ticket image button
b. Edit Location
c. Edit Time
4. Orientation Change

Sequence 5-6:
1. Click on an element in list
2. Click on Edit icon
3. Click on Ticket image button
4. Click on:
a. Select Category
b. Select Color
5. Orientation Change

Sequence 7-9:
1. Click on Add button
2. Click on Create Pass
3. Click on:
a. Add Barcode
b. Edit Location
c. Edit Time

Sequence 10:
1. Click on Add Button
2. Click on Scan for pkpass files
3. Orientation Change

**Comment by max-te** [comment]
This is particularly annoying since barcode scanner forces landscape mode and the change back to PassAndroid cancels editing the barcode.

**Comment by simontb** [comment]
I agree that this is very inconvenient and should be fixed. Meanwhile you could disable auto-rotate before scanning a barcode. The barcode scanner will still force landscape mode, but PassAndroid will remain in portrait mode after returning and thus stay in the edit screen and all changes will be kept.

Please let me know in case this doesn't work for you.

**Comment by max-te** [comment]
No, fixed rotation does not work for me: When returning from Barcode Scanner to PassAndroid, PassAndroid changes rotation very briefly. The only method which works is keeping the phone in landscape mode through the entire process.
